securedrop/manage.py

```diff
@@ -57,56 +59,152 @@ def _stop_test_rqworker():
     os.kill(get_pid_from_pidfile(WORKER_PIDFILE), signal.SIGTERM)
 
 
-def start():
-    import config
-    source_rc = subprocess.call(['start-stop-daemon',
-                                 '--start',
-                                 '-b',
-                                 '--quiet',
-                                 '--pidfile',
-                                 config.SOURCE_PIDFILE,
-                                 '--startas',
-                                 '/bin/bash',
-                                 '--',
-                                 '-c',
-                                 'cd /vagrant/securedrop && python source.py'])
-    journo_rc = subprocess.call(['start-stop-daemon',
-                                 '--start',
-                                 '-b',
-                                 '--quiet',
-                                 '--pidfile',
-                                 config.JOURNALIST_PIDFILE,
-                                 '--startas',
-                                 '/bin/bash',
-                                 '--',
-                                 '-c',
-                                 'cd /vagrant/securedrop && python journalist.py'])
-
-    if source_rc + journo_rc == 0:
-        print "The web application is running, and available on your Vagrant host at the following addresses:"
-        print "Source interface:     localhost:8080"
-        print "Journalist interface: localhost:8081"
-    else:
-        print "The web application is already running.  Please use './manage.py restart' to stop and start again."
+def run():
```

Comment:
We currently use `build` for build artifacts from the build scripts, but I think putting it all into a module is a reasonable idea. Perhaps `mangement` or `mgmt`? Then ./manage.py can do `from mgmt import run` to make the `run` function available.